examples/webhooks-manager/pages/index.tsx

```diff
@@ -249,6 +249,7 @@ export default function Main({
       <div className="cs1 ce12">
         <div className="feedback">
           <a
+            target="_top"
```

Comment:
What does this solve?

Reply:
Seems like chrome prevents opening these links inside the iframe. This will make the browser open the link outside the iframe, at the "top" and that should trigger opening the email client.